src/main/resources/anniv5Resources/localization/eng/runicpack/Powerstrings.json

```
@@ -0,0 +1,15 @@
+{
+  "${ModID}:ObeliskPower": {
+    "NAME": "Power of the Obelisk",
+    "DESCRIPTIONS": [
+      "When you change #yStances, Channel a random #yOrb.",
```

Base game powers (and cards) that channel orbs use "1" instead of "a" for the singular case (see e.g. Static Discharge, Storm).
```suggestion
      "When you change #yStances, Channel 1 random #yOrb.",
```